apps/api/schema.py

```diff
@@ -29,6 +29,25 @@ class BearerScheme(TokenScheme):
     target_class = "apps.api.permissions.BearerTokenAuthentication"
 
 
+class EmbeddedWidgetScheme(OpenApiAuthenticationExtension):
+    target_class = "apps.api.authentication.EmbeddedWidgetAuthentication"
+    name = "embedKeyAuth"
+    match_subclasses = True
+    priority = -1
+
+    def get_security_definition(self, auto_schema):
+        return {
+            "type": "apiKey",
+            "in": "header",
+            "name": "X-Embed-Key",
+            "description": (
+                "Embedded widget authentication token. Validates the widget token from the experiment channel "
+                "configuration and authenticates embedded widget requests. Requires chatbot_id (experiment public_id) "
+                "in the request body for /api/chat/start/ or from session data for subsequent requests."
```

Comment:
nit: remove 'experiment' terminology which we are phasing out
```suggestion
                "Embedded widget authentication token. Validates the widget token for the channel "
                "configuration and authenticates embedded widget requests. Requires chatbot_id "
                "in the request body for /api/chat/start/ or from session data for subsequent requests."
```